Python/cp/bulb_switcher.py

```diff
@@ -0,0 +1,19 @@
+# Bulb Swither problem
+# link to problem: https://leetcode.com/problems/bulb-switcher/
+
+
+def bulbSwitcher(n):
```

Comment:
Add proper comments to explain your approach, this will help beginners a lot. You may format your code according to our [python code template](https://github.com/TesseractCoding/NeoAlgo/blob/master/Python/Code_Template.md).

Reply:
okay